pycoal/mineral.py

```diff
@@ -142,10 +153,100 @@ def SAM(library, image_file_name, classified_file_name, scores_file_name=None, t
                 'map info': image.metadata.get('map info')
             })
 
+def avngDNN(image_file_name, classified_file_name, model_file_name, class_names=None, scores_file_name=None, in_memory=False):
```

Comment:
This function name is pretty specific... has the code been tested against any AVIRIS-C imagery or only AVIRIS-NG?

Reply:
No at all. The nem of the function is really misleading, because one can input a model that works with any shape of data, either AVIRIS-C, AVIRIS-NG or even a resample approach. I'm going to change it.

Reply:
Please go ahead and change the function name to something more descriptive.